tests/api/test_content_api.py

```diff
@@ -581,3 +581,27 @@ def test_search(self):
         r = self.api_client.get(url, format="json")
         self.assertEqual(r.status_code, 200)
         self.assertEqual(len(r.data["results"]), 4)
+
+
+class TestContentResolveAPI(BaseAPITestCase):
+
+    def test_resolve_url(self):
+        make_content(id=123)
+        r = self.api_client.get(reverse("content-resolve-list"),
+                                dict(url="/r/123"), format="json")
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(123, r.data['id'])
```

Comment:
Yeah i'll throw the params on.
The longer URLs are property-specific and not defined in django-bulbs, but might make sense to mock one up.